deployments/ansible/roles/collector/tasks/linux_install.yml

```diff
@@ -37,6 +37,29 @@
     mode: 0600
   notify: "restart splunk-otel-collector"
 
+- name: Read the default config
+  slurp:
+    path: "{{splunk_otel_collector_config}}"
+  register: defaultconfig
+  when: splunk_config_override != ''
+
+- name: Extract the default config
+  set_fact:
+    updated_config: "{{ defaultconfig['content'] | b64decode | from_yaml }}"
+  when: splunk_config_override != ''
+
+- name: Update the default config
+  set_fact:
+    updated_config: "{{ updated_config | combine(splunk_config_override, recursive=True, list_merge='append_rp') }}"
+  when: splunk_config_override != ''
+
+- name: Copy the updated config
+  copy:
+    content: '{{ updated_config | to_nice_yaml }}'
+    dest: "{{splunk_otel_collector_config}}"
+  when: splunk_config_override != ''
+  notify: "restart splunk-otel-collector"
+
```

Comment:
Reuse this block across windows and linux playbook. probably should be moved to another file

Reply:
Yes, but we need to move this into two different files because copy module is different for Windows and Linux.

Reply:
you can reuse the common part only

Reply:
Ok, Thanks.